src/pages/dev_tap/index.tsx

```diff
@@ -0,0 +1,137 @@
+import {
```

Comment:
Is this supposed to mock the collection view? Or the tap view?

Reply:
It's supposed to be a mock for tapping, i.e. by just being able to tap a button and get a jubmoji